sb-cb-ui-utils-v2/library/sunbird-cb/utils-v2/src/lib/pipes/pipe-public-URL/pipe-public-URL.pipe.ts

```diff
@@ -9,9 +9,12 @@ export class PipePublicURL implements PipeTransform {
     this.environment = environment
   }
   transform(value: string): any {
-    const mainUrl = value && value.split('/content').pop() || ''
-    const finalURL = `${this.environment.contentHost}/${this.environment.contentBucket}/content${mainUrl}`
-    return value ? finalURL : ''
+    if (value.includes("karmayogi")) {
+      const mainUrl = value && value.split('/content').pop() || ''
+      const finalURL = `${this.environment.contentHost}/${this.environment.contentBucket}/content${mainUrl}`
+      return value ? finalURL : ''
+    } else {
+      return value
```

Comment:
Please add comments, why you are doing this change

Reply:
For the CIOS content, the image is a s3 URL. we can access that directly